tests/Integration/Behaviour/Features/Scenario/Product/update_pack.feature

```diff
@@ -85,32 +86,34 @@ Feature: Add product to pack from Back Office (BO)
     And I add product "product3" with following information:
       | name[en-US] | summerstreet |
       | type        | virtual      |
+    And I add new image "image3" named "logo.jpg" to product "product3"
     And I add product "product4" with following information:
       | name[en-US] | winterstreet |
       | type        | virtual      |
+    And I add new image "image4" named "logo.jpg" to product "product4"
     And product "product3" type should be virtual
     And product "product4" type should be virtual
     When I update pack "productPack2" with following product quantities:
       | product  | quantity |
       | product3 | 3        |
       | product4 | 20       |
     Then product "productPack2" type should be pack
-    And pack productPack2 should contain products with following quantities:
-      | product  | quantity |
-      | product3 | 3        |
-      | product4 | 20       |
+    And pack "productPack2" should contain products with following details:
+      | product  | combination | name             | quantity | image url                                          |
+      | product3 |             | summerstreet     | 3        | http://myshop.com/img/p/{image3}.jpg |
```

Comment:
Wow I didn't expect you to handle the image URL for each product, only one would have been enough but it's great

However I'm sorry but now it's the `no_picture` case which is not handled any more 😅 Can you please leave at least one product and one combination without an image associated to check that the no picture fallback works in both cases?

Reply:
I know it's already handled in other unrelated tests like duplicate feature (at least for standard products) But It's just so that this feature handles all cases related to packs

Reply:
product has no more image. So we will have the two case ... with or without image !